modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -1050,11 +1050,11 @@ When defining an input of type XML, there are a few optional parameters you can
 [cols="2,1,1,2", options="header"]
 |===
 |Parameter |Type |Default |Description
-|`optimizeFor`| string | speed | specifies the strategy to be used by the reader. Posible values = memory/speed
-|`nullValueOn`|string | 'empty' |If a tag with empty or blank text should be read as null.
-|`indexedReader`|boolean | true | Picks which reader modality to use. The indexed reader is faster but uses up a greater amount of memory, whilst the unindexed reader is slower but uses up less memory
-|`maxEntityCount`|integer | 1 | Limits the number of times that an entity can be referenced within the XML code. This is included to guard against https://en.wikipedia.org/wiki/Billion_laughs[denial of service attacks].
-|`externalEntities`|boolean | false | Defines if references to entities that are defined in a file outside the XML are accepted as valid. It's recommended to avoid these for https://www.owasp.org/index.php/XML_External_Entity_(XXE)_Processing[security reasons] as well.
+|`nullValueOn`|string | `empty` |If a tag with empty or blank text should be read as null.
+|`indexedReader`|boolean | `true` | Picks which reader modality to use. The indexed reader is faster but uses up a greater amount of memory, whilst the unindexed reader is slower but uses up less memory
+| `escapeCR` | Boolean | `false` | Whether to escape a carriage return (CR). Valid options: `true` or `false`
+|`maxEntityCount`|integer | `1` | Limits the number of times that an entity can be referenced within the XML code. This is included to guard against https://en.wikipedia.org/wiki/Billion_laughs[denial of service attacks].
+|`externalEntities`|boolean | `false` | Defines if references to entities that are defined in a file outside the XML are accepted as valid. It's recommended to avoid these for https://www.owasp.org/index.php/XML_External_Entity_(XXE)_Processing[security reasons] as well.
```

Comment:
What does the "as well" refer to? "as well" as what? I would remove it.

Reply:
I think we should add a comment that it is available since 4.2.1